src/lightkurve/interact.py

```diff
@@ -257,6 +274,32 @@ def make_lightcurve_figure_elements(lc, lc_source, ylim_func=None):
     return fig, vertical_line
 
 
+def _add_nearby_tics_if_tess(tpf, source, tooltips):
+    tic_id = tpf.meta['TICID']
```

Comment:
I'm sure you are aware, but this line of course raises a `KeyError` for a Kepler TPF right now, e.g. this fails:
```python
lk.search_targetpixelfile("Kepler-10", mission="Kepler").download().interact_sky()
```